app/assets/stylesheets/checkin.scss

```diff
@@ -0,0 +1,7 @@
+.disable-field-set {
+  background: gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.